package.json

```diff
@@ -18,20 +18,22 @@
   "main": "./dist/index.js",
   "types": "./dist/index.d.ts",
   "files": [
-    "dist/**/*"
+    "dist/**/*.map",
```

Comment:
This change is to avoid including `tsconfig.tsbuildinfo`, which is a new "cache" file `tsc` uses when you use Project References and the [`tsc --build` command](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-4.html#faster-subsequent-builds-with-the---incremental-flag).